src/renderer/components/cluster-manager/bottom-bar.tsx

```diff
@@ -45,29 +45,36 @@ export class BottomBar extends React.Component {
       return null;
     }
 
+    items.sort(function sortLeftPositionFirst(a, b) {
+      return a.components?.position?.localeCompare(b.components?.position);
+    });
+
     return (
-      <div className={styles.extensions}>
+      <>
         {items.map((registration, index) => {
           if (!registration?.item && !registration?.components?.Item) {
             return null;
           }
 
           return (
-            <div className={styles.item} key={index}>
+            <div
+              className={cssNames(styles.item, {
+                [styles.onLeft]: registration.components?.position == "left",
```

Comment:
nit, `components` will be defined here.

Reply:
Removed optional at first, but then reverted it back - this is more for tests to pass since we're testing different APIs in there.

Reply:
Ah sounds good